.github/actions/aws-ci/action.yml

```diff
@@ -14,7 +14,13 @@ inputs:
     description: 'AWS account type'
     required: false
     default: single-account
-
+  cloudbeat-config:
+    description: 'Full path to a cloudbeat configuration file, e.g. "deploy/aws/cloudbeat-aws.yml"'
+    required: true
+  pytest-keyword-expression:
+    description: 'Keyword expression to use as a pytest -k flag valyu'
+    required: true
+    default: 'aws'
```

Comment:
It would be better to update to one of the existing options: `default: cspm_aws` or `default: asset_inventory_aws`. Alternatively, this will not be relevant if new actions are created.

Reply:
Resolved by having a separate action for asset inventory.